SConstruct

```diff
@@ -536,6 +545,17 @@ for variant_path in variant_paths:
     env = main.Clone()
     env['BUILDDIR'] = variant_path
 
+    cdb_tool = SCons.Tool.FindTool(['compilation_db'], env)
+
+    if cdb_tool:
+        env['COMPILATIONDB_USE_ABSPATH'] = True
+        env.Tool(cdb_tool)
+
+        cdb_path = f"{variant_path}/compile_commands.json"
+        cdb_build_target = env.CompilationDatabase(cdb_path)
+
+        Alias(f"{variant_path}/compile_commands", cdb_build_target)
```

Comment:
Is the alias required? Can we have the target be `compile_commands.json` instead of `compile_commands`?

I ask this because all other targets are explicit files. (e.g., `gem5.opt` or the path to a `.o` file)

Reply:
> Can we have the target be compile_commands.json instead of compile_commands?

We can change the target to `build/{ISA}/compile_commands.json` instead of `build/{ISA}/compile_commands`

Moreover, we no need to use `Alias` in that case. 
Currently,  `Alias` is used to force the target name to `build/{ISA}/compile_commands`.

I will reflect this change is PR.